src/tr1d1um/http.go

```diff
@@ -0,0 +1,287 @@
+package main
+
+import (
+	"net/http"
+	"strings"
+	"encoding/json"
+	"regexp"
+	"time"
+	"bytes"
+	"fmt"
+	"github.com/Comcast/webpa-common/wrp"
+)
+var paramRegex *regexp.Regexp
+
+/*
+type ConversionHandler struct {
+	//todo add //loggers
+	timeOut time.Duration
+	targetUlr string
+}
+
+func (sh ConversionHandler) ServeHTTP(response http.ResponseWriter, request *http.Request) {
+	_ , err := HttpRequestToWRP(request, ioutil.ReadAll)
+	if err != nil { //something is wrong in the received request
+		response.WriteHeader(http.StatusBadRequest)
+		response.Write([]byte(err.Error()))
+		return
+	}
+
+	//todo forespard converted wrp request to targetUrl with timeout
+
+	/*
+	encodedWrp, err :=  attemptEncoding(wrpRequest, wrp.MustEncode)
+
+	if err != nil { // cannot put 100% blame on request
+		response.WriteHeader(http.StatusInternalServerError)
+		response.Write([]byte(err.Error()))
+		return
+	}
+	var payload bytes.Buffer
+
+	r := bufio.NewReader(&payload)
+
+	//todo actually post stuff to other server
+	req, err := http.NewRequest("POST", sh.targetUlr,r)
+
+	if err != nil {
+		response.WriteHeader(http.StatusInternalServerError)
+		response.Write([]byte(err.Error()))
+		return
+	}
+}
+*/
+func ConversionHandler(resp http.ResponseWriter, req *http.Request){
+	wdmp := new(WDMP)
+
+	//read in parameters and command
+	wdmp.Names = strings.Split(req.FormValue("names"), ",")
+	wdmp.Command = req.Method
+
+	//Get payload for transfer
+	payload, err := json.Marshal(wdmp)
+
+	if err != nil {
+		return
+	}
+
+	//todo: place it into wrp?
+	wrpMessage := wrp.Message{}
+	wrpMessage.Type = wrp.SimpleEventMessageType
```

Comment:
The wrpMessage should be a `wrp.SimpleRequestResponseMessageType` type since you will expect a response.

Reply:
@schmidtw That is correct. I have fixed that. Currently, I have rearranged a few things and I am writing unit tests for the functions I have.